### ISSUE TITLE
conda 4.5.0-py36_0 package gives error: CondaHTTPError: HTTP 000 CONNECTION FAILED for url 

### ISSUE THREAD

**Issue by afonit** [conda/conda#7150]
Using a fresh install of:
Anaconda3-5.1.0-Windows-x86_64.exe
on Windows 7.

**I can install packages fine using conda 4.4.10-py36_0**,
`but` after I update to conda 4.5.0-py36_0 I get errors:
```
$ conda update -n base conda
Solving environment: ...working... done

## Package Plan ##

  environment location: C:\Users\some_user_name\AppData\Local\Continuum\anaconda3

  added / updated specs:
    - conda


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    conda-4.5.0                |           py36_0         1.0 MB

The following packages will be UPDATED:

    conda: 4.4.10-py36_0 --> 4.5.0-py36_0

Proceed ([y]/n)?
```

When hit 'y' to install it, then I go to install another package - I get the below error message, and am no longer able to install packages:

```
$ conda install pandas
Solving environment: ...working... failed

# >>>>>>>>>>>>>>>>>>>>>> ERROR REPORT <<<<<<<<<<<<<<<<<<<<<<

`$ C:\Users\some_user_name\AppData\Local\Continuum\anaconda3\Scripts\conda install pandas`

  environment variables:
             ACLOCAL_PATH=C:\Program Files\Git\mingw64\share\aclocal;C:\Program
                          Files\Git\usr\share\aclocal
                 CIO_TEST=<not set>
                CLASSPATH=C:\Program Files\Teradata\Client\14.10\JMSAXSMOD\;C:\Program
                          Files\Teradata\Client\14.10\JMSAXSMOD\jmsam.jar;C:\Program Files
                          (x86)\Teradata\Client\14.10\JMSAXSMOD\;C:\Program Files
                          (x86)\Teradata\Client\14.10\JMSAXSMOD\jmsam.jar
               CONDA_ROOT=C:\Users\some_user_name\AppData\Local\Continuum\anaconda3
     DATACONNECTORLIBPATH=C:\Program Files\Teradata\Client\14.10\bin\;C:\Program Files
                          (x86)\Teradata\Client\14.10\bin\
                  EXEPATH=C:\Program Files\Git
                 HOMEPATH=\Users\some_user_name
                 INFOPATH=C:\Program Files\Git\usr\local\info;C:\Program
                          Files\Git\usr\share\info;C:\Program Files\Git\usr\info;C:\Program
                          Files\Git\share\info
                  MANPATH=C:\Program Files\Git\mingw64\share\man;C:\Program
                          Files\Git\usr\local\man;C:\Program Files\Git\usr\share\man;C:\Program
                          Files\Git\usr\man;C:\Program Files\Git\share\man
            ORIGINAL_PATH=C:\Program Files\Git\mingw64\bin;C:\Program Files\Git\usr\bin;C:\Users
                          \some_user_name\bin;C:\ProgramData\Oracle\Java\javapath;C:\Program Files
                          (x86)\RSA SecurID Token Common;C:\Program
                          Files\Teradata\client\14.10\Teradata Parallel
                          Transporter\bin64;C:\Program Files\Teradata\client\14.10\Teradata
                          Parallel Transporter\msg64;C:\Program Files
                          (x86)\Teradata\client\14.10\Teradata Parallel
                          Transporter\bin;C:\Program Files (x86)\Teradata\client\14.10\Teradata
                          Parallel Transporter\msg;C:\Program
                          Files\Teradata\Client\14.10\pp2_64;C:\Program
                          Files\Teradata\Client\14.10\bin64;C:\Program Files
                          (x86)\Teradata\Client\14.10\pp2;C:\Program
                          Files\Teradata\Client\14.10\AXSMOD;C:\Program Files
                          (x86)\Teradata\Client\14.10\AXSMOD;C:\Program
                          Files\Teradata\Client\14.10\bin;C:\Program Files
                          (x86)\Teradata\Client\14.10\bin;C:\Program
                          Files\Teradata\Client\14.10\CLIv2;C:\Program Files
                          (x86)\Teradata\Client\14.10\CLIv2;C:\Program
                          Files\Teradata\Client\14.10\ODBC Driver for Teradata
                          nt-x8664\Lib;C:\Program Files (x86)\Teradata\Client\14.10\ODBC Driver
                          for Teradata\Lib;C:\Program Files\Teradata\Client\14.10\Shared ICU
                          Libraries for Teradata\lib;C:\Program Files
                          (x86)\Teradata\Client\14.10\Shared ICU Libraries for Teradata\lib;C:\W
                          INDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System3
                          2\WindowsPowerShell\v1.0;C:\Windows\CCM;C:\Program Files
                          (x86)\Teradata\Client\14.10\tdwallet\nt-i386;C:\Program
                          Files\Teradata\Client\14.10\tdwallet\nt-x8664;C:\Program Files
                          (x86)\PuTTY;C:\Program Files (x86)\Symantec\VIP Access
                          Client;C:\PostgreSQL\pg96\bin;C:\Program Files (x86)\Microsoft SQL
                          Server\100\Tools\Binn\VSShell\Common7\IDE;C:\Program Files
                          (x86)\Microsoft SQL Server\100\Tools\Binn;C:\Program Files
                          (x86)\Microsoft SQL Server\100\DTS\Binn;C:\Program
                          Files\MIT\Kerberos\bin;C:\Program Files (x86)\Windows Kits\8.1\Windows
                          Performance Toolkit;C:\Program Files (x86)\WebEx\Productivity
                          Tools;C:\Program Files\Git\cmd;C:\Users\some_user_name\AppData\Local\Contin
                          uum\anaconda3;C:\Users\some_user_name\AppData\Local\Continuum\anaconda3\Lib
                          rary\mingw-w64\bin;C:\Users\some_user_name\AppData\Local\Continuum\anaconda
                          3\Library\usr\bin;C:\Users\some_user_name\AppData\Local\Continuum\anaconda3
                          \Library\bin;C:\Users\some_user_name\AppData\Local\Continuum\anaconda3\Scri
                          pts;C:\Users\some_user_name\AppData\Local\Programs\MiKTeX
                          2.9what\miktex\bin\x64
                     PATH=C:\Users\some_user_name\bin;C:\Program Files\Git\mingw64\bin;C:\Program
                          Files\Git\usr\local\bin;C:\Program Files\Git\usr\bin;C:\Program
                          Files\Git\usr\bin;C:\Program Files\Git\mingw64\bin;C:\Program Files\Gi
                          t\usr\bin;C:\Users\some_user_name\bin;C:\ProgramData\Oracle\Java\javapath;C
                          :\Program Files (x86)\RSA SecurID Token Common;C:\Program
                          Files\Teradata\client\14.10\Teradata Parallel
                          Transporter\bin64;C:\Program Files\Teradata\client\14.10\Teradata
                          Parallel Transporter\msg64;C:\Program Files
                          (x86)\Teradata\client\14.10\Teradata Parallel
                          Transporter\bin;C:\Program Files (x86)\Teradata\client\14.10\Teradata
                          Parallel Transporter\msg;C:\Program
                          Files\Teradata\Client\14.10\pp2_64;C:\Program
                          Files\Teradata\Client\14.10\bin64;C:\Program Files
                          (x86)\Teradata\Client\14.10\pp2;C:\Program
                          Files\Teradata\Client\14.10\AXSMOD;C:\Program Files
                          (x86)\Teradata\Client\14.10\AXSMOD;C:\Program
                          Files\Teradata\Client\14.10\bin;C:\Program Files
                          (x86)\Teradata\Client\14.10\bin;C:\Program
                          Files\Teradata\Client\14.10\CLIv2;C:\Program Files
                          (x86)\Teradata\Client\14.10\CLIv2;C:\Program
                          Files\Teradata\Client\14.10\ODBC Driver for Teradata
                          nt-x8664\Lib;C:\Program Files (x86)\Teradata\Client\14.10\ODBC Driver
                          for Teradata\Lib;C:\Program Files\Teradata\Client\14.10\Shared ICU
                          Libraries for Teradata\lib;C:\Program Files
                          (x86)\Teradata\Client\14.10\Shared ICU Libraries for Teradata\lib;C:\W
                          INDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System3
                          2\WindowsPowerShell\v1.0;C:\Windows\CCM;C:\Program Files
                          (x86)\Teradata\Client\14.10\tdwallet\nt-i386;C:\Program
                          Files\Teradata\Client\14.10\tdwallet\nt-x8664;C:\Program Files
                          (x86)\PuTTY;C:\Program Files (x86)\Symantec\VIP Access
                          Client;C:\PostgreSQL\pg96\bin;C:\Program Files (x86)\Microsoft SQL
                          Server\100\Tools\Binn\VSShell\Common7\IDE;C:\Program Files
                          (x86)\Microsoft SQL Server\100\Tools\Binn;C:\Program Files
                          (x86)\Microsoft SQL Server\100\DTS\Binn;C:\Program
                          Files\MIT\Kerberos\bin;C:\Program Files (x86)\Windows Kits\8.1\Windows
                          Performance Toolkit;C:\Program Files (x86)\WebEx\Productivity
                          Tools;C:\Program Files\Git\cmd;C:\Users\some_user_name\AppData\Local\Contin
                          uum\anaconda3;C:\Users\some_user_name\AppData\Local\Continuum\anaconda3\Lib
                          rary\mingw-w64\bin;C:\Users\some_user_name\AppData\Local\Continuum\anaconda
                          3\Library\usr\bin;C:\Users\some_user_name\AppData\Local\Continuum\anaconda3
                          \Library\bin;C:\Users\some_user_name\AppData\Local\Continuum\anaconda3\Scri
                          pts;C:\Users\some_user_name\AppData\Local\Programs\MiKTeX
                          2.9what\miktex\bin\x64;C:\Program
                          Files\Git\usr\bin\vendor_perl;C:\Program Files\Git\usr\bin\core_perl
          PKG_CONFIG_PATH=C:\Program Files\Git\mingw64\lib\pkgconfig;C:\Program
                          Files\Git\mingw64\share\pkgconfig
             PSMODULEPATH=C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules\
               PYTHONPATH=C:\Users\some_user_name\AppData\Local\Continuum\;C:\Program Files\Microsoft
                          VS Code\bin
       REQUESTS_CA_BUNDLE=<not set>
            SSL_CERT_FILE=<not set>

     active environment : None
       user config file : C:\Users\some_user_name\.condarc
 populated config files : C:\Users\some_user_name\.condarc
          conda version : 4.5.0
    conda-build version : 3.4.1
         python version : 3.6.4.final.0
       base environment : C:\Users\some_user_name\AppData\Local\Continuum\anaconda3  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/win-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/free/win-64
                          https://repo.anaconda.com/pkgs/free/noarch
                          https://repo.anaconda.com/pkgs/r/win-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/pro/win-64
                          https://repo.anaconda.com/pkgs/pro/noarch
                          https://repo.anaconda.com/pkgs/msys2/win-64
                          https://repo.anaconda.com/pkgs/msys2/noarch
          package cache : C:\Users\some_user_name\AppData\Local\Continuum\anaconda3\pkgs
                          C:\Users\some_user_name\AppData\Local\conda\conda\pkgs
       envs directories : C:\Users\some_user_name\AppData\Local\Continuum\anaconda3\envs
                          C:\Users\some_user_name\AppData\Local\conda\conda\envs
                          C:\Users\some_user_name\.conda\envs
               platform : win-64
             user-agent : conda/4.5.0 requests/2.18.4 CPython/3.6.4 Windows/7 Windows/6.1.7601
          administrator : False
             netrc file : None
           offline mode : False


V V V V V V V V V V V V V V V V V V V V V V V V V V V V V V V

CondaHTTPError: HTTP 000 CONNECTION FAILED for url <https://repo.anaconda.com/pkgs/free/win-64/repodata.json.bz2>
Elapsed: -

An HTTP error occurred when trying to retrieve this URL.
HTTP errors are often intermittent, and a simple retry will get you on your way.

If your current network has https://www.anaconda.com blocked, please file
a support request with your network engineering team.

SSLError(MaxRetryError('HTTPSConnectionPool(host=\'repo.anaconda.com\', port=443): Max retries exceeded with url: /pkgs/free/win-64/repodata.json.bz2 (Caused by SSLError(SSLError("bad handshake: Error([(\'SSL routines\', \'ssl3_get_server_certificate\', \'certificate verify failed\')],)",),))',),)

A reportable application error has occurred. Conda has prepared the above report.
If submitted, this report will be used by core maintainers to improve
future releases of conda.
Would you like conda to send this report to the core maintainers?
[y/N]:
```

As far as repeatably - this will happen every time after updating to that conda package.

Note, before we could work fine with only these urls being unblocked:
conda.anaconda.org
repo.continuum.io
repo.anaconda.com

As the error message says:
```
If your current network has https://www.anaconda.com blocked, please file
a support request with your network engineering team.
```
I am working with IT to see if we can unblock that, but wanted to make sure that url dependency was just added in this version of conda.

**Comment by afonit** [comment]
I have confirmed with our IT support team, and they have added in www.anaconda.com

But the error noted in the first post still persists.

**Comment by dhirschfeld** [comment]
You could try:
```
conda config --set ssl_verify false 
```

**Comment by afonit** [comment]
@dhirschfeld , looking at other threads it would appear that doing the command you specified is not recommended.

I am trying to figure out from the conda group if the upgrade in conda from:
4.4.10-py36_0 --> 4.5.0-py36_0
changed url dependencies and if this is a bug.

**Comment by kalefranz** [comment]
In a web browser, can you open https://repo.anaconda.com/pkgs/free/win-64/repodata.json?

**Comment by afonit** [comment]
@kalefranz , yes I can.
I am assuming you are saying it should work then, so I will risk upgrading conda again and seeing if it works.

**Comment by afonit** [comment]
@kalefranz , I could browse to that page.
so I went ahead and upgraded conda and I not get the same error when trying to install packages.
```
(base) C:\Users\some_user_name>conda update -n base conda
Solving environment: done

## Package Plan ##

  environment location: C:\Users\some_user_name\AppData\Local\Continuum\anaconda3

  added / updated specs:
    - conda


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    conda-4.5.0                |           py36_0         1.0 MB

The following packages will be UPDATED:

    conda: 4.4.10-py36_0 --> 4.5.0-py36_0

Proceed ([y]/n)? y


Downloading and Extracting Packages
conda 4.5.0: ########################################################## | 100%
Preparing transaction: done
Verifying transaction: done
Executing transaction: done

(base) C:\Users\some_user_name>block should really be the equivalent of
'block' is not recognized as an internal or external command,
operable program or batch file.

(base) C:\Users\some_user_name>
```



and here is the error now:
```
CondaHTTPError: HTTP 000 CONNECTION FAILED for url <https://repo.anaconda.com/pk
gs/pro/win-64/repodata.json.bz2>
Elapsed: -

An HTTP error occurred when trying to retrieve this URL.
HTTP errors are often intermittent, and a simple retry will get you on your way.


If your current network has https://www.anaconda.com blocked, please file
a support request with your network engineering team.

SSLError(MaxRetryError('HTTPSConnectionPool(host=\'repo.anaconda.com\', port=443
): Max retries exceeded with url: /pkgs/pro/win-64/repodata.json.bz2 (Caused by
SSLError(SSLError("bad handshake: Error([(\'SSL routines\', \'ssl3_get_server_ce
rtificate\', \'certificate verify failed\')],)",),))',),)

A reportable application error has occurred. Conda has prepared the above report
.
If submitted, this report will be used by core maintainers to improve
future releases of conda.
Would you like conda to send this report to the core maintainers?
[y/N]:
```

**Comment by kalefranz** [comment]
Can you continue with

    conda install pandas -k

?

**Comment by afonit** [comment]
Yes.

**Comment by dhirschfeld** [comment]
It look like a SSL error so turning off verification may be the only way to get it to work.

Likely your IT dept is inspecting your traffic and injecting self-signed certs. Your browser works because it uses the Windows cert store but requests uses certifi? so knows nothing about the injected certs and so doesn't verify then.

... that's my hunch anyway.

**Comment by afonit** [comment]
@dhirschfeld , Yes indeed, that is why we had these urls whitelisted:
```
conda.anaconda.org
repo.continuum.io
repo.anaconda.com
www.anaconda.com
```
Which has worked for us for about a year.  However it is not now working.  So we are trying to figure out what has changed between these two versions of the conda package manager:
```
4.4.10-py36_0 --> 4.5.0-py36_0
```

Since the 4.4 works fine, but the 4.5 does not.

**Comment by afonit** [comment]
Switching between an external network and the internal one:
* it works on the external
* it does not work on the internal one.

If it was the antivirus would it not interrupt it in both places since it is same laptop in both scenarios - or is it your understanding that sometimes antivirus software is doing something special when on a corporate network vs an external one?

**Comment by shytikov** [comment]
`conda 4.5.12` workaround is not working for me...

**Comment by msarahan** [comment]
If you're on Windows, please see #8046

**Comment by lampfot** [comment]
SOLVED!

If you're on windows and on a company network using a proxy, here is what worked for me after trying all other things which didn't work at all:

[https://github.com/ContinuumIO/anaconda-issues/issues/3292#issuecomment-533489446](https://github.com/ContinuumIO/anaconda-issues/issues/3292#issuecomment-533489446)

>Solution:
>
>To configure proxy. This would allow you to install packages.
>
>    Start menu search for “env”
>   Select “Edit Environment Variables for your account”
>    Press “New..”
>   and add two variables “http_proxy” and “https_proxy” both with the same value http://proxy->XX:XXX
>
>Notice URL is http (not https) for both http_proxy and https_proxy. The name has https but the url does not.
>
>the "http://proxy-XX:XXX" need to be the route proxy of your company.
